app/config/settings.py

```diff
@@ -356,6 +356,7 @@ def strtobool(val) -> bool:
     "grandchallenge.workstation_configs",
     "grandchallenge.policies",
     "grandchallenge.favicons",
+    "grandchallenge.ai_website",
```

Comment:
App names stick around and I think that it's important to get this right. Can we think of a better name? Products?